client/components/domains/registrant-extra-info/with-contact-details-validation.jsx

```diff
@@ -128,6 +128,10 @@ export default function WithContactDetailsValidation( tld, WrappedComponent ) {
 					tld: tld,
 					contactDetails: this.props.contactDetails,
 				} );
+				// Add an error at the top level so the form knows something went wrong
+				// We can't attribute it to a specific field, so we'll use the
+				// empty string as a placeholder of the right type that should
+				// never appear as a field name.
```

Comment:
Did you really pass up a perfect opportunity to use a [`Symbol`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol)?!

☝️ Mostly joking 😅 I'm pretty sure we'd have compatibility issues and don't want to investigate the polyfill situation. This seems a good solution 👍

Reply:
On another day I'd be keen to try out new stuff, let me know if you find a good excuse :)

It looks like we have had `Symbol` in the code base for nearly a year without compatibility problems - ([client/my-sites/domains/domain-management/list/constants.js](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/domains/domain-management/list/constants.js) since #3931), and the non-collision is perfect, but this worries me a little:

`JSON.stringify( { [Symbol()]: "^o^" } ) // =>  "{}"`